kernelci/api/helper.py

```diff
@@ -279,10 +279,17 @@ def create_job_node(self, job_config, input_node,
             'path': input_node['path'] + [job_config.name],
             'group': job_config.name,
             'artifacts': {},
+            'jobfilter': input_node['jobfilter'],
             'data': {
                 'kernel_revision': input_node['data']['kernel_revision'],
             },
         }
+        # if jobfilter not null, verify if job_config.name exist in jobfilter
+        if input_node['jobfilter']:
+            if job_config.name not in input_node['jobfilter']:
```

Comment:
```suggestion
        if jobfilter:
            if job_config.name not in jobfilter:
```

According to the above comment

Reply:
Updated